css/parser/tests/recovery/delim-token/bar/output.swc-stderr

```diff
@@ -1,6 +0,0 @@
-error: Expected Declaration value
```

Comment:
Same here.

Reply:
Yes, it is valid output now, we need to move them in other place